Tasks/AzureRmWebAppDeploymentV3/operations/ReleaseAnnotationUtility.ts

```diff
@@ -45,12 +45,22 @@ function getReleaseAnnotation(isDeploymentSuccess: boolean): {[key: string]: any
  
     let releaseAnnotationProperties = {
         "Label": isDeploymentSuccess ? "Success" : "Error", // Label decides the icon for release annotation
-        "Deployment Uri": getDeploymentUri()
+        "Deployment Uri": getDeploymentUri(),
+        "BuildNumber": getPipelineVariable("Build.BuildNumber"),
+        "BuildRepositoryName": getPipelineVariable("Build.Repository.Name"),
+        "BuildRepositoryProvider": getPipelineVariable("Build.Repository.Provider"),
+        "SourceBranch": getPipelineVariable("Build.SourceBranch"),
+        "ReleaseId": getPipelineVariable("Release.ReleaseId"),
+        "ReleaseDescription": getPipelineVariable("Release.ReleaseDescription"),
+        "ReleaseDefinitionName": getPipelineVariable("Release.ReleaseDescription"),
+        "ReleaseEnvironmentName": getPipelineVariable("Release.EnvironmentName"),
+        "ReleaseRequestedFor": getPipelineVariable("Release.RequestedForId") || getPipelineVariable("Release.RequestedFor"),
+        "ReleaseWebUrl": getPipelineVariable("Release.ReleaseWebURL"),
```

Comment:
Yes. The idea is if the variable doesn't exist, it will be ignored. This is to bring the current release annotation to be inline with what we are having in our blades (we have our own release annotation pipeline task that does just like this).